x/evm/keeper/attest.go

```diff
@@ -77,33 +77,33 @@ func (k Keeper) attestRouter(ctx sdk.Context, q consensus.Queuer, msg consensust
 	ctx, writeCache := ctx.CacheContext()
 	defer func() {
 		if err != nil {
-			logger.WithError(err).Error("failed to attest. Skipping writeback.")
+			logger.With(err).Error("failed to attest. Skipping writeback.")
 			return
 		}
 		writeCache()
 	}()
 
 	consensusMsg, err := msg.ConsensusMsg(k.cdc)
 	if err != nil {
-		logger.WithError(err).Error("failed to cast to consensus message")
+		logger.With(err).Error("failed to cast to consensus message")
 		return err
 	}
 
 	evidence, err := k.findEvidenceThatWon(ctx, msg.GetEvidence())
 	if err != nil {
 		if errors.Is(err, ErrConsensusNotAchieved) {
-			logger.WithError(err).Error("consensus not achieved")
+			logger.With(err).Error("consensus not achieved")
```

Comment:
Why this change? Please revert this.

Reply:
Reverted the change and made the changes in the  util/liblog as well.